<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>워밍업 프로그램</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="getPriceLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>시세조회</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="getPriceTextEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="getPriceButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>조회</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>매수</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="buyTextEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="buyLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>매수 금액</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="sellTextEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sellLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>매도 금액</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sellButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>매도</string>
    </property>
   </widget>
   <widget class="QLabel" name="getPriceErrorLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buyErrorLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sellErrorLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>490</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
